modules/storages/app/workers/storages/manage_nextcloud_integration_cron_job.rb

```diff
@@ -35,5 +35,13 @@ class ManageNextcloudIntegrationCronJob < Cron::CronJob
     queue_with_priority :low
 
     self.cron_expression = '*/5 * * * *'
+
+    def self.ensure_scheduled!
```

Comment:
This method is used by a rake task for scheduling cron_jobs. So, the job is still registered on boot time. Then the rake task calls this method for all registered jobs.